scripts/analyze.sh

```diff
@@ -304,7 +304,23 @@ if [ $? -eq 0 ] || check_config_file ZJS_SENSOR; then
     >&2 echo Using module: Sensor
     MODULES+=" -DBUILD_MODULE_SENSOR"
     echo "export ZJS_SENSOR=y" >> zjs.conf.tmp
-    if [ $BOARD = "arduino_101" ]; then
+    if [ $DEV = "ashell" ]; then
```

Comment:
This should go into prj.conf.dev or something, shouldn't it?

@brianjjones, Most/all of these are set by analyze.sh for the non-ashell case, when appropriate.